src/main/java/br/com/deveficiente/bolaoapi/services/team/Team.java

```diff
@@ -28,11 +33,37 @@
     @Getter
     private LocalDate foundation;
 
+    @ManyToMany(mappedBy = "teams")
+    @Getter
+    private Set<Championship> championships;
```

Comment:
Sugiro só fazer o bidirecional quando realmente for utilizar. O que não é o caso agora.

Reply:
:+1: